### DISCUSSION TITLE
org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing column [full_name] in table

### DISCUSSION THREAD

**Discussion by kumawatanupriya** [Project-Books/book-project#683] [Q&A]
@knjk04 Getting this exception while running the project, 
I did flyway migrate.
full_name is not added in DB, and I don't see any migration file for that. Please let me know if I need to do something from my side.

**Comment by knjk04** [comment]
Do you have the script https://github.com/Project-Books/book-project/blob/0.2.0/backend/src/main/resources/db/migration/V031__ALTER_AUTHOR_MAKE_NAME_FULL.sql? If not, I'd recommend bringing your branch level with 0.2.0
